server/templates/staff/course/assignment/assignment.new.html

```diff
@@ -25,33 +25,74 @@
 
     {% include 'alerts.html' %}
     <div class="row">
-      <div class="col-md-6">
+      <div class="col-md-12">
         <!-- Default box -->
           <div class="box">
             <div class="box-body">
-              {% call forms.render_form(form, action_url="", action_text='Create Assignment', class_='form') %}
-                  {{ forms.render_field(form.display_name, label_visible=true, placeholder='Hog', type='text') }}
-                  {{ forms.render_field(form.name, label_visible=true, value=current_course.offering + '/', placeholder='cal/cs61a/fa16/proj01', type='text') }}
+              {% call forms.render_form_bare(form, action_url="", class_='form new-assign-form wizard') %}
+                <h3>Name</h3>
+                <fieldset>
+                  {{ forms.render_field(form.display_name, label_visible=true, placeholder='Hog',
+                                        required="required", type='text', id="display-name") }}
+                  {{ forms.render_field(form.name, label_visible=true, value=current_course.offering + '/',
+                                        placeholder='cal/cs61a/fa16/proj01', type='text', id="assignment-offering") }}
```

Comment:
The placeholder should be `cal/cs61a/fa16/hog` using our new assignment name format